cms/djangoapps/contentstore/tests/test_contentstore.py

```diff
@@ -6,6 +6,7 @@
 import shutil
 import lxml
 
+from bs4 import BeautifulSoup
```

Comment:
since we're already importing `lxml`, any objections to using `lxml.etree` instead?